multi-node/aws/README.md

```diff
@@ -1,210 +1,28 @@
-
-You can also now check the `my-cluster` asset directory into version control if you desire. The contents of this directory are your reproducible cluster assets. Please take care not to commit the `my-cluster/credentials` directory, as it contains your TLS secrets. If you're using git, the `credentials` directory will already be ignored for you.
-
-## Route53 Host Record (optional)
-
-`kube-aws` can optionally create an A record for the controller IP in an existing hosted zone.
-
-Edit the `cluster.yaml` file:
-
-```yaml
-externalDNSName: my-cluster.staging.core-os.net
-createRecordSet: true
-hostedZone: staging.core-os.net
-```
-
-If `createRecordSet` is not set to true, the deployer will be responsible for making externalDNSName routable to the controller IP after the cluster is created.
-
-## Validate your cluster assets
-
-The `validate` command check the validity of the cloud-config userdata files and the cloudformation stack description:
-
-```sh
-$ kube-aws validate
-```
-
-## Create a cluster from asset directory
-
-```sh
-$ kube-aws up
-```
-
-This command can take a while.
-
-## Access the cluster
-
-```sh
-$ kubectl --kubeconfig=kubeconfig get nodes
-```
-
-It can take some time after `kube-aws up` completes before the cluster is available. Until then, you will have a `connection refused` error.
-
-## Export your cloudformation stack
-
-```sh
-$ kube-aws up --export
-```
-
-## Development
-
-### Build
-
-Run the `./build` script to compile `kube-aws` locally.
-
-This depends on having:
-* golang >= 1.5
-
-The compiled binary will be available at `bin/kube-aws`.
-
-### Run Unit Tests
-
-```sh
-go test $(go list ./... | grep -v '/vendor/')
-```
-
-### Modifying templates
-
-The various templates are located in the `pkg/config/templates/` folder of the source repo. `go generate` is used to pack these templates into the source code. In order for changes to templates to be reflected in the source code:
-
-```sh
-go generate ./pkg/config
-```
-
-This command is run automatically as part of the `build` script.
-
-### Useful Resources
-
-The following links can be useful for development:
-
-- [AWS CloudFormation resource types](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-template-resource-type-ref.html)
-
-## Contributing
-
-Submit a PR to this repository, following the [contributors guide](../../CONTRIBUTING.md).
-The documentation is published from [this source](../../Documentation/kubernetes-on-aws.md).
-
 
+	|
+	|                *----------------*
+	|              *                    *
+	|            *                        *
+	|          *                            *
+	|        *                                *
+	|        |    ___  _____   ____    ____   |
+	|        |   |       |    |    |  |    |  |
+	|        |   |___    |    |    |  |____|  |
```

Comment:
Not sure I agree with this ASCII art. You will still be able to contribute to this repo...